plaso/parsers/plist_plugins/ios_carplay.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""Plist parser plugin for Car Play App plist files.
+   It contains history of opened apps on Car Play App.
+"""
+
+from dfdatetime import posix_time
+
+from plaso.containers import plist_event
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.parsers import plist
+from plaso.parsers.plist_plugins import interface
+
+
+class IOSCarPlayPlugin(interface.PlistPlugin):
+  """Plist parser plugin for IOS Car Play App plist files.
+
+  """
+
+  NAME = 'ios_carplay'
+  DATA_TYPE = 'ios:carplay:history'
+  DATA_FORMAT = 'iOS Car Play App plist file'
+
+  PLIST_PATH_FILTERS = frozenset([
+    interface.PlistPathFilter('com.apple.CarPlayApp.plist')])
+
+  PLIST_KEYS = frozenset(['CARRecentAppHistory'])
+
+  # pylint: disable=arguments-differ
+  def _ParsePlist(self, parser_mediator, match=None, **unused_kwargs):
+    """Extract relevant Car Play App entries.
+    """
+    recent_app_history = match.get('CARRecentAppHistory', {})
+    for parameter, value in recent_app_history.items():
+      event_data = plist_event.PlistTimeEventData()
```

Comment:
Define a data type specific EventData object

Reply:
Hi @joachimmetz what is `a data type specific EventData object`? Can you point an example from another plist parser?

Reply:
I see that plist parsers still use a generic PlistTimeEventData, I'll fix this